demos/TicTacToe/Classes/main_menu_scene.cc

```diff
@@ -704,11 +661,27 @@ void MainMenuScene::update(float /*delta*/) {
     } else if (current_state_ == kAuthState) {
       // Sign out logic, adding auth screen.
       auth_background_->setVisible(true);
+      // Pauses all event touch listeners & then resumes the ones attached to
+      // auth_background_.
+      Director::getInstance()
```

Comment:
It would be clearner to store the result from Director::getInstnace()->getEventDispatcher() in a variable and then invoke the pause and resume methods from the variable.  Fewer lines.  Better Pizza.  Pappa Johns.

Reply:
Yeah, I mean, who doesn't?